### PR TITLE
Include one new FAQ Question under "General"

### PR DESCRIPTION
**Pull request by avsavsavs**
Included the question "Why is Cowswap a META DEX Aggregator?". The question goes from line 176 to 183

### CODE REVIEW

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions on a variety of on-chain liquidity.
+            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool /path offers
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:179]:
```suggestion
            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool/path offers 
```

**Review by W3stside — APPROVED** [review]
minor comment

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions on a variety of on-chain liquidity.
```
Comment (bh2smith) [inline src/custom/pages/Faq/index.tsx:178]:
```suggestion
            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions for a variety of on-chain liquidity sources.
```

File: src/custom/pages/Faq/index.tsx [hunk 3]
```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
```
Comment (bh2smith) [inline src/custom/pages/Faq/index.tsx:176]: I might reword this to something like
```suggestion
          <h3 id="why-is-cowswap-a-meta-dex-aggregator">What makes CowSwap a "Meta" DEX aggregator?</h3>
```

File: src/custom/pages/Faq/index.tsx [hunk 4]
```diff
@@ -173,6 +173,15 @@ export default function Faq() {
...
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions on a variety of on-chain liquidity.
+            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool /path offers 
+            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators.  In addition to that, before
```
Comment (bh2smith) [inline src/custom/pages/Faq/index.tsx:180]: Nit: Ideally each sentence is on its own line.

File: src/custom/pages/Faq/index.tsx [hunk 5]
```diff
@@ -173,6 +173,15 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">Why is CowSwap a Meta DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions on a variety of on-chain liquidity.
+            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool /path offers 
+            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators.  In addition to that, before 
+            finding the best trading price for a trade from the available on-chain liquidity, Gnosis Protocol first checks if it 
+            can find a coincidence of wants within the existing batch to offer an even better price than any pool can.
```
Comment (bh2smith) [inline src/custom/pages/Faq/index.tsx:182]: This is an example where I want to make a suggested change for the sentence but have to suggest over multiple lines.

```suggestion
            finding the best price for a trade from available on-chain liquidity, Gnosis Protocol first seeks a coincidence of wants within the existing batch to offer an even better price than any pool can.
```

**Review by bh2smith — CHANGES_REQUESTED** [review]
Looks pretty good. Made a few suggested changes.

File: src/custom/pages/Faq/index.tsx [hunk 6]
```diff
@@ -173,6 +173,14 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">What makes CowSwap a "Meta" DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions for a variety of on-chain liquidity sources.
+            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool/path offers 
+            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators.  In addition to that, before
```
Comment (bh2smith) [inline src/custom/pages/Faq/index.tsx:180]:
```suggestion
            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators. In addition to that, before 
```

**Review by bh2smith — APPROVED** [review]
Looks really good content wise. However, I would very much like to see each sentence on its own dedicated line for future maintainability. Of course this is not mandatory so the changes are approved. Just a strong suggestion about good practices.

**Comment by bh2smith** [comment]
It appears that mergify is configured to merge as soon as there is >1 approval automatically... I will remember this for next time since you didn't have a chance to address my optional suggestion here.

**Comment by W3stside** [comment]
> It appears that mergify is configured to merge as soon as there is >1 approval automatically... I will remember this for next time since you didn't have a chance to address my optional suggestion here.

oops, yeah he's greedy. we sometimes make "verbal approvals" for moments like this, if it's not a major required change

**Review by anxolin — COMMENTED** [review]
Nice!

I would highlight that one powerful thing CowSwap can do, and why is different to DexAggregators, is that the liquidity sources used can be determined at execution time, so there's no need to pre-commit to a particular execution path that might not be available anymore. This gives great flexibility and can make Cow trades succeed where others aggregators would fail and make you loose gas.

**Comment by alfetopito** [comment]
@bh2smith I've created a new PR doing what you suggested https://github.com/gnosis/cowswap/pull/759
